go/install/install_windows.go

```diff
@@ -328,3 +331,28 @@ func getCachedPackageModifyString(log Log) (string, error) {
 	log.Debug("getCachedPackageModifyString: " + `SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall` + bundleKey + "displayName " + displayName + ", publisher " + publisher)
 	return "", errors.New("no cached package path found")
 }
+
+// StartUpdateIfNeeded starts to update the app if there's one available. It
+// calls `updater check` internally so it ignores the snooze.
+func StartUpdateIfNeeded(ctx context.Context, log logger.Logger) error {
```

Comment:
Windows part LGTM, here - this was just moved over, right?

Reply:
Moved over, and added the `Wait()`. But yeah, it's mostly the same code for Windows. I still need to test it on Windows after the build is out and see if update actually works.